lang/nim/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                nim
+version             1.4.4
+revision            0
+license             MIT
+categories          lang
+platforms           darwin
+supported_archs     i386 x86_64 ppc64
```

Comment:
You would only set `supported_archs` if you know for sure that a port only works with a given subset of architectures.

In this case, it looks like `nim` supports the whole hog, including **arm64**. So remove this and the right thing should get passed automatically as per the local architecture.

Reply:
I was going off of their [README](https://github.com/nim-lang/Nim#compiling) which lists only x86, x86_64, and ppc64 for macOS. That’s for stable/tested versions only, so I suppose it does technically build for arm64 but it’s not officially supported by the Nim project (yet). What’s the call here in this case? Do we list *all* archs even if not officially supported, or do we stick with what upstream says?

Reply:
Ah, in that case we should stick with upstream, but still remember to open things up (remove `supported_archs`) if the next version has official support for ARM64.